kani-driver/src/cbmc_output_parser.rs

```diff
@@ -681,4 +713,17 @@ mod tests {
             serde_json::from_str(prop_id_string);
         let _prop_id = prop_id_result.unwrap();
     }
+
+    #[test]
+    fn check_trace_value_deserialization_works() {
+        let data = r#"{
+            "binary": "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
nit: can you generate this string using padding format? Something like:

```rust
   let data = format!(r#"{{
            "binary": "{:0>1000}",
            "data": "0",
            "name": "integer",
            "type": "unsigned __CPROVER_bitvector[960]",
            "width": 960
        }}"#, 0);
```

Just replace the 1000 by whatever number you want.

Reply:
Done.